src/doc/rustc/src/platform-support/bpf-unknown-none.md

```diff
@@ -0,0 +1,123 @@
+# `bpf*-unknown-none`
+
+**Tier: 3**
+
+* `bpfeb-unknown-none` (big endian)
+* `bpfel-unknown-none` (little endian)
+
+Targets for the [eBPF virtual machine][ebpf].
+
+## Target maintainers
+
+- [@dave-tucker](https://github.com/dave-tucker)
+- [@vadorovsky](https://github.com/vadorovsky)
```

Comment:
@alessandrod @tamird Should I add you as well?

Reply:
Yep.